README.rst

```diff
@@ -360,6 +360,10 @@ Choose from the list of available fixers:
                         Remove trailing commas in list
                         function calls.
 
+* **lower_case_cast** [symfony]
```

Comment:
I see, if misread.. but than again, we have `spaces_cast` and `short_bool_cast`. 
Adding `lowercase_cast` and `short_scalar_cast` would be more inline of not using plural. I don't feel much for one or the other, but it would be nice to be consistent. It is also not upped in https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/889 to rename the current cast fixer names. Let me know what you think (cc @keradus) and I rename accordingly.